src/v2/Apps/Collect/Routes/Collection/Components/CollectionArtworksFilter.tsx

```diff
@@ -22,6 +22,7 @@ import { ArtistNationalityFilter } from "v2/Components/ArtworkFilter/ArtworkFilt
 import { MaterialsFilter } from "v2/Components/ArtworkFilter/ArtworkFilters/MaterialsFilter"
 import { PartnersFilter } from "v2/Components/ArtworkFilter/ArtworkFilters/PartnersFilter"
 import { ArtistsFilter } from "v2/Components/ArtworkFilter/ArtworkFilters/ArtistsFilter"
+import { __internal__useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
```

Comment:
**chore**: as far as I can see it is not used and can be safely removed

Reply:
good catch @dimatretyak, thought I removed that! Weird that tsc or pre commit/push hooks didn't catch that!

Reply:
this is because there is a `__` in name

Reply:
gotcha! :) Thanks!